File: car-factory/src/database/enums/car.enum.ts
export enum Color{
    WHITE = 1,
    BLACK = 2,
    GREY = 3,
    RED = 4,
    BLUE = 5
}

export enum CarModel{
    HATCH = 1,
    SEDAN = 2,
    SUV = 3,
    PICKUP = 4
}

export enum Model{
    Yaris,
    Corolla,
    Camry,
    CorollaCross,
    RAV4,
    Hilux,
    SW4,

}

export const ColorLabel = new Map<number, string>([
    [Color.WHITE, 'Branco'],
    [Color.BLACK, 'Preto'],
    [Color.GREY, 'Cinza'],
    [Color.RED, 'Vermelho'],
    [Color.BLUE, 'Azul'],
  ]);

File: CarFactory.Stock/CarFactory.Stock/Models/Enum/CarEnum.cs
using System.ComponentModel;

namespace CarFactory.Stock.Models.Enum;

public enum Model
{
    Yaris,
    Corolla,
    Camry,
    CorollaCross,
    RAV4,
    Hilux,
    SW4,
}

public enum CarModel
{
    HATCH = 1,
    SEDAN = 2,
    SUV = 3,
    PICKUP = 4
}

public enum Color
{
    [Description("Branco")]
    WHITE = 1,
    [Description("Preto")]
    BLACK = 2,
    [Description("Cinza")]
    GREY = 3,
    [Description("Vermelho")]
    RED = 4,
    [Description("Azul")]
    BLUE = 5
}